### PR TITLE
feat(ProgressBarStepped): create component

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1572)
Specs: [Link](https://www.figma.com/file/3IO1EsGDeBRiFdF23mrvuj/%F0%9F%94%B8-ProgressBar-Specs?type=design&node-id=0-1&mode=design&t=FgMz6OTwS5KxEKRK-0)

### CODE REVIEW

**Review by yceballost — DISMISSED** [review]

File: src/__stories__/progress-bar-story.tsx [hunk 1]
```diff
@@ -11,25 +11,57 @@ export default {
     },
 };
 
-type Args = {
+type ProgressBarStoryArgs = {
     reverse: boolean;
     progressPercent: number;
     color: 'default' | 'red';
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/progress-bar-story.tsx]: done

File: src/__screenshot_tests__/__image_snapshots__/progress-bar-screenshot-test-tsx-progress-bar-stepped-color-default-1-snap.png
Comment (yceballost) [inline src/__screenshot_tests__/__image_snapshots__/progress-bar-screenshot-test-tsx-progress-bar-stepped-color-default-1-snap.png:1]: Why does the screenshot have this space on the top?

**Review by yceballost — APPROVED** [review]

File: src/__screenshot_tests__/__image_snapshots__/progress-bar-screenshot-test-tsx-progress-bar-stepped-color-default-1-snap.png
Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/progress-bar-screenshot-test-tsx-progress-bar-stepped-color-default-1-snap.png:1]: good catch. fixed

**Review by pladaria — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**